compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaBasedOwner.skiko.kt

```diff
@@ -217,7 +216,9 @@ internal class SkiaBasedOwner(
 
     override val autofill: Autofill? get() = null
 
-    override val viewConfiguration: ViewConfiguration = DefaultViewConfiguration(density)
+    override val viewConfiguration: ViewConfiguration = platform.viewConfiguration(
```

Comment:
If ViewConfiguration depends on a mutable density, maybe we can make do something like:

```kotlin
override val viewConfiguration: ViewConfiguration by derivedStateOf { ViewConfiguration(..., density)}
```
So viewConfiguration will get updated as soon as density.

Reply:
viewConfiguration creates outside of `@Composable` functions.
Do derivedStateOf work's well in this case?

Reply:
Unfortunately it not updates with density changes

Reply:
> density is controlled by the client of the ComposeScene.

Sent my comment. Review was in "pending state", so it is in the middle )

Reply:
Done